health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryRelationCreate.js

```
@@ -214,14 +468,14 @@ const onConfirmClick=()=>{
                                 <div key={`boundary-${index}`}>
                                     <div style={{ fontWeight: "600", fontSize: "1.2rem" }}>
                                         {/* {item?.boundaryType} */}
-                                        {`${t((defaultHierarchyType + "_" + item?.boundaryType).toUpperCase().replace(/\s+/g, "_"))}`}
+                                        {`${t((defaultHierarchyType + "_" + item?.boundaryType).toUpperCase())}`}
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)